libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1301,67 +1293,68 @@ void AP_InertialSensor::update(void)
 
             if (_startup_ms == 0) {
                 _startup_ms = AP_HAL::millis();
-            } else if (AP_HAL::millis()-_startup_ms > 2000) {
+            } 
+            else if (AP_HAL::millis() - _startup_ms > 2000) {
```

Comment:
This change is wrong, goes against style guide. There are more places like this.